pandas/core/frame.py

```diff
@@ -3569,6 +3569,32 @@ def transpose(self, *args, copy: bool = False) -> DataFrame:
 
     @property
     def T(self) -> DataFrame:
+        """
+        The transpose of the DataFrame.
+
+        Returns
+        -------
+        DataFrame
+            The transposed DataFrame.
+
+        Examples
+        --------
+        >>> df = pd.DataFrame({'col1': [1, 2], 'col2': [3, 4]})
+        >>> df
+           col1  col2
+        0     1     3
+        1     2     4
+
+        >>> df.T
+              0  1
+        col1  1  2
+        col2  3  4
+
+
```

Comment:
There should be only one blank line here.

And it seems more appropriate that the order is `Returns`, `See also`, `Examples`.

Reply:
Thank you - agreed.  I have made those changes.